test/tests/functional/pbs_mom_dynamic_resource_ext.py

```diff
@@ -60,98 +61,27 @@ def create_mom_dyn_resc(self, resc_name,
                                         validate=True)
         self.scheduler.add_resource(resc_name)
 
-    def test_t1(self):
-        """
-        Test for host level consumable size resource
-        with string mom dynamic value
-        """
-        resc_name = "foo"
-        self.create_mom_dyn_resc(resc_name, "size", "nh", "/bin/echo abc")
-
-        a = {'scheduling': 'True'}
-        self.server.manager(MGR_CMD_SET, SERVER, a)
-
-        # Submit a job that requests mom dynamic resource
-        attr = {"Resource_List." + resc_name: 1}
-        j = Job(TEST_USER, attrs=attr)
-        jid = self.server.submit(j)
-
-        # The job shouldn't run
-        self.server.expect(JOB, {'job_state': 'Q'}, id=jid)
-
-        # Check for the expected log message in scheduler logs
-        c = "Insufficient amount of resource: foo (1kb != (null))"
```

Comment:
Did you file this bug?  It is a product bug.

Reply:
Thanks for filing the bug!

Reply:
@bhroam and @dinesh1306 should not we keep the test but comment the log message due to PP-1263? And these tests as reference to PP-1263.

Reply:
The tests have value as negative test cases.  The problem with including them now is that depending how PP-1263 is fixed, the tests will have to change.  Probably the best way the fix PP-1263 is to reject the resource at the time we query the mom.  The test as written will not work for that.  We might want to wait until PP-1263 is fixed before adding these tests.

Reply:
@bhroam you are right that functionality might change which will change the test but that should not be a reason not to include this test for 18.x. We have found an issue where negative values has caused scheduler to behave differently. At least having this test around for 18.x will help catching such issues (Today it is not crashing it might crash tomorrow with the same test). (PBS behavior might change but objective of the test would remain the same).

Reply:
The test as it stands tests for the buggy behavior.  I do not think we should include that.  If we try and fix the test, we'd need to know what the non-buggy behavior is.  We won't know that until the bug is fixed.

Reply:
okay thanks